mm2src/coins/coin_errors.rs

```diff
@@ -17,7 +17,9 @@ pub enum ValidatePaymentError {
     UnexpectedPaymentState(String),
     Transport(String),
     // Transaction has wrong properties, for example, it has been sent to a wrong address
-    WrongPaymentTx(String),
+    InvalidTx(String),
```

Comment:
Sure. After your [#1457 (comment)](https://github.com/KomodoPlatform/atomicDEX-API/pull/1457#discussion_r961780722) comment I also implemented my own `ValidatePaymentError` some weeks ago independently. Then yesterday during the merge I mixed the two implementations cherry-picking my favorite from both but I'll take back my own changes since there's already so much discussion behind it.